app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -381,8 +378,9 @@ class TopicLessonsFragmentTest {
 
   private fun scrollToPosition(position: Int) {
     onView(withId(R.id.story_summary_recycler_view)).perform(
-      scrollToPosition<RecyclerView.ViewHolder>(
-        position
+      actionOnItemAtPosition<RecyclerView.ViewHolder>(
```

Comment:
There are many other files in the whole codebase where we are using `scrollToPosition<RecyclerView.ViewHolder>(position)`. 

So, as for this file only, if we bring consistency by using `actionOnItemAtPosition<RecyclerView.ViewHolder>()` everywhere, it's also good, but we do need a helper class which is something single point where we can look for when to use which scrolling way.

Reply:
Done.